usaspending_api/api_contracts/contracts/autocomplete/Glossary.md

```diff
@@ -2,39 +2,68 @@
 
 This endpoint returns glossary autocomplete data for submitted text snippet.
 
-
-+ Parameters
-    + search_text: `ab` (required, string)
-        The text snippet that you are trying to autocomplete using a glossary term.
-    + limit: `10` (optional, integer)
-        Maximum number to return
-
 ### List Autocomplete Glossary [POST]
 
 + Request (application/json)
 
     + Attributes
 
-        + search_text: `ab` (required, string)
+        + `search_text`: `aw` (required, string)
         The text snippet that you are trying to autocomplete using a glossary term.
-        + limit: `10` (optional, number)
+        + `limit`: 10 (optional, number)
         Maximum number to return
-        
+
 + Response 200 (application/json)
-    + Attributes (GlossaryAutocompleteObject)
+    + Attributes
+        + `results` (required, array[string], fixed-type)
+        + `count` (required, number)
+        + `search_text` (required, string)
+        + `matched_terms` (required, array[GlossaryListing], fixed-type)
+
+    + Body
+
+            {
+                "results": [
+                    "Award",
+                    "Award Amount",
+                    "Award ID",
+                    "Award Type",
+                    "Awarding Agency",
+                    "Awarding Office",
+                    "Awarding Sub-Agency",
+                    "Current Award Amount",
+                    "Multiple Award Schedule (MAS)",
+                    "Parent Award Identification (ID) Number"
+                ],
+                "search_text": "aW",
+                "count": 10,
+                "matched_terms": [
+                    {
+                        "term": "Award",
+                        "slug": "award",
+                        "data_act_term": null,
+                        "plain": "Money the federal government has promised to pay a recipient. Funding may be awarded to a company, organization, government entity (i.e., state, local, tribal, federal, or foreign), or individual. It may be obligated (promised) in the form of a contract, grant, loan, insurance, direct payment, etc.",
```

Comment:
This seem volatile for an API contract

Reply:
Brian, this is an example JSON response. It isn't part of the schema or response enforcement